jpa/org.eclipse.persistence.jpa.jpql/src/main/java/org/eclipse/persistence/jpa/jpql/WordParser.java

```diff
@@ -178,7 +178,6 @@ public WordType getWordType() {
      */
     public boolean isArithmeticSymbol(char character) {
         return character == '>' ||
-               character == '!' ||
```

Comment:
is removal of `!` character here intended change?

Reply:
Sorry it was typo there. It's fixed in now.